Dockerfile.erb

```diff
@@ -37,6 +27,10 @@ RUN set -ex \
 	&& tar -xzvf $BITCOIN_DIST -C /usr/local --strip-components=1 --exclude=*-qt \
 	&& rm bitcoin*
 
+RUN apt-get autoremove -qqy --purge ca-certificates wget gpg dirmngr \
+	&& apt-get clean \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
It is indeed totally useless and counterproductive... I was expecting image size to decrease but it did actually increased as you expected - I have to read a bit more on Docker to get more understanding